dbt/models/ccao/docs.md

```diff
@@ -1,3 +1,11 @@
+# aprval_reascd
+
+{% docs table_aprval_reascd %}
+Reason codes and descriptions for changes to assessed value.
```

Comment:
**suggestion:** Let's add a quick note here to make clear that this is just a static Parquet file in the warehouse.